pkg/sidecar/sidecar.go

```diff
@@ -183,6 +232,131 @@ func (s *Sidecar) dumpBundles(svidResponse *workloadapi.X509Context) error {
 	return nil
 }
 
+func (s *Sidecar) writeJSON(fileName string, certs map[string]interface{}) {
+	file, err := json.Marshal(certs)
+	if err != nil {
+		s.config.Log.Errorf("Unable to parse certs: %v", err)
```

Comment:
it is going to fail to marshal, but try to store an empty []byte that is going to remvoe actual file...
is that ok?

Reply:
Well, if it fails to renew the bundles the helper will erase the current and still valid bundles right? So i think it's incorrect behavior. Maybe it should end without writing anything.